Demo/Swift_Demo/ViewController/ExampleTableViewController.swift

```diff
@@ -73,4 +73,13 @@ class ExampleTableViewController: BaseViewController, UITableViewDataSource, UIT
 
         return cell!
     }
+
+    // ✅ FIX: Reset bottom contentInset to 0
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        
+        if let tableView = self.view.subviews.first(where: { $0 is UITableView }) as? UITableView {
```

Comment:
Please create an @IBOutlet reference of UITableView from storyboard and then use it directly instead of finding it in subview.